<launch>
  <include file="$(find terrain_analysis)/launch/terrain_analysis.launch" />
  
  <node pkg="terrain_analysis" type="planeAnalysis" name="planeAnalysis" output="screen" required="true">
    <param name="intensity_threshold" type="double" value="0.05" />
    <param name="plane_angle_threshold" type="double" value="60.0" />
    <param name="remain_points_threshold" type="int" value="50" />
  </node>

  <!-- <node launch-prefix="nice" pkg="rviz" type="rviz" name="rvizGA" args="-d $(find terrain_analysis)/rviz/plane.rviz" respawn="true"/> -->

</launch>
